test/e2e/dynamic_provisioning_test.go

```diff
@@ -581,8 +581,11 @@ var _ = ginkgo.Describe("[blob-csi-e2e] Dynamic Provisioning", func() {
 				Cmd: "echo 'hello world' > /mnt/test-1/data && grep 'hello world' /mnt/test-1/data",
 				Volumes: []testsuites.VolumeDetails{
 					{
-						ClaimSize:    "10Gi",
-						MountOptions: []string{},
+						ClaimSize: "10Gi",
+						MountOptions: []string{
+							"-o allow_other",
+							"--virtual-directory=true", // blobfuse2 mount options
```

Comment:
cc @cvvz I have added a blobfuse2 specific mount options in e2e test, that could ensure the blobfuse2 test is correct even it's using blobfuse v1 mount actually.